UI/frontend-plugins/aja-output-ui/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+project(aja-output-ui)
+
+if(APPLE)
+	find_library(COCOA Cocoa)
+	include_directories(${COCOA})
+endif()
+
+if(UNIX AND NOT APPLE)
+	find_package(X11 REQUIRED)
+	link_libraries(${X11_LIBRARIES})
+	include_directories(${X11_INCLUDE_DIR})
+
+	find_package(Qt5X11Extras REQUIRED)
+endif()
+
+set(aja-output-ui_HEADERS
+	${aja-output-ui_HEADERS}
+	../../qt-wrappers.hpp
+	../../properties-view.hpp
+	../../properties-view.moc.hpp
+	../../vertical-scroll-area.hpp
+	../../double-slider.hpp
+	../../slider-ignorewheel.hpp
+	../../combobox-ignorewheel.hpp
+	../../spinbox-ignorewheel.hpp
+		AJAOutputUI.h
+		aja-ui-main.h
+	../../../plugins/aja/aja-ui-props.hpp
+	../../../plugins/aja/aja-enums.hpp
+	)
+set(aja-output-ui_SOURCES
+	${aja-output-ui_SOURCES}
+	../../qt-wrappers.cpp
+	../../properties-view.cpp
+	../../vertical-scroll-area.cpp
+	../../double-slider.cpp
+	../../slider-ignorewheel.cpp
+	../../combobox-ignorewheel.cpp
+	../../spinbox-ignorewheel.cpp
+		AJAOutputUI.cpp
+		aja-ui-main.cpp
+	)
+
+set(aja-output-ui_UI
+	${aja-output-ui_UI}
+	forms/output.ui)
+
+if(WIN32)
+	set(MODULE_DESCRIPTION "OBS AJA Output UI")
+	configure_file(${CMAKE_SOURCE_DIR}/cmake/winrc/obs-module.rc.in aja-output-ui.rc)
+	list(APPEND aja-output-ui_SOURCES
+		aja-output-ui.rc)
+endif()
+
+if(APPLE)
+	set(aja-output-ui_PLATFORM_LIBS
+		${COCOA})
+endif()
+
+qt5_wrap_ui(aja-output-ui_UI_HEADERS
+	${aja-output-ui_UI})
+
+add_library(aja-output-ui MODULE
+	${aja-output-ui_HEADERS}
+	${aja-output-ui_SOURCES}
+	${aja-output-ui_UI_HEADERS}
+	)
+
+if (APPLE)
+	target_compile_definitions(aja-output-ui PUBLIC
+		AJAMac
+		AJA_MAC)
+elseif(WIN32)
+	target_compile_definitions(aja-output-ui PUBLIC
+		AJA_WINDOWS
+		_WINDOWS
+		WIN32
+		MSWindows)
+elseif(UNIX AND NOT APPLE)
+	target_compile_definitions(aja-output-ui PUBLIC
+		AJA_LINUX
+		AJALinux)
+endif()
+
+target_include_directories(aja-output-ui PUBLIC
+	../../../plugins/aja/sdk/ajalibraries)
+
+target_link_libraries(aja-output-ui
+	${frontend-tools_PLATFORM_LIBS}
+	obs-frontend-api
+	Qt5::Widgets
+	libobs)
+
+if(UNIX AND NOT APPLE)
+	target_link_libraries(aja-output-ui
+		Qt5::X11Extras)
```

Comment:
I dont see `QX11Info` being included so this can probably just be removed?

Reply:
This AJA Output UI code started with the Decklink Output UI code as a base. I'll take a look at changes to the Decklink code where the Qt dependencies have changed since work on the AJA plugin was started.